style/section2.css

```diff
@@ -0,0 +1,125 @@
+*{
```

Comment:
Try not to sue plain css, use a css framework or a UI Kit like Chakra, Material UI,  Bootstrap, Tailwind CSS / Headless UI